microsoft-edge/devtools-guide-chromium/json-viewer/json-viewer.md

```diff
@@ -0,0 +1,67 @@
+---
+title: View formatted JSON
+description: How to use the JSON viewer to view formatted and highlighted JSON responses in a browser tab.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/04/2023
+---
+# View formatted JSON
+
+Use the **JSON viewer** to automatically format and highlight JSON responses and files in browser tabs.
+
+The **JSON viewer** changes the returned JSON data to make it easier to read. Sometimes a web server responds to HTTP requests by returning data encoded as JSON.  JSON data can be difficult to read when it's formatted as a single long, concatenated line of text. The same can occur when opening a JSON file from disk.
+
+The **JSON viewer** improves readability of JSON data in several ways:
+
+*  The JSON syntax is highlighted with different colors.
+*  Object properties are displayed on their own lines and indented.
+*  Objects can be collapsed or expanded.
```

Comment:
```suggestion
*  Objects can be collapsed or expanded.
*  The JSON viewer shows dark and light themes.
```
Is this the best place and format to cover that the JSON viewer shows dark and light themes?
See context of that [comment](https://github.com/MicrosoftDocs/edge-developer/pull/2378#discussion_r1065812523).